yt_dlp/extractor/banbye.py

```diff
@@ -4,9 +4,12 @@
 from .common import InfoExtractor
 from ..utils import (
     InAdvancePagedList,
+    determine_ext,
     format_field,
+    int_or_none,
     traverse_obj,
```

Comment:
if applying my formats suggestion, we'll need `join_nonempty`:
```suggestion
    format_field,
    int_or_none,
    join_nonempty,
    traverse_obj,
```